src/bentoml/_internal/cloud/bentocloud.py

```diff
@@ -544,7 +544,9 @@ def pull_model(model_tag: Tag):
                                 continue
                             p = Path(member.name)
                             if p.parent != Path("."):
-                                temp_fs.makedirs(str(p.parent), recreate=True)
+                                temp_fs.makedirs(
+                                    str(p.parent.as_posix()), recreate=True
```

Comment:
```suggestion
                                    p.parent.as_posix(), recreate=True
```
`as_posix()` is a string itself, no need the `str()`